src/components/StarRating.ts

```diff
@@ -36,15 +38,29 @@ export class StarRating extends Component<StarRatingProps, {}> {
         this.fractions = readOnly ? 2 : 1;
         this.stop = this.props.maximumStars;
 
-        return createElement("div", {
-            className: classNames("widget-star-rating", this.props.className),
-            style: this.props.style
-        },
+        const customStyle: object | undefined = this.props.starSize === "custom"
+            ? { fontSize: `${this.props.starSizeCustom}px` }
+            : undefined;
+        const EmptyStar = createElement("div", {
+            className: `glyphicon glyphicon-star-empty widget-star-rating-empty ` +
+                `widget-star-rating-font-${this.props.starSize}`,
+            style: customStyle
+        });
+        const FullStar = createElement("span", {
+            className: `glyphicon glyphicon-star widget-star-rating-font-${this.props.starSize} ` +
+            `widget-star-rating-full-${this.props.widgetColor}`,
+            style: customStyle
+        });
+
+        return createElement("div",
```

Comment:
Just a question: Why do we have this additional wrapper div? cant we move classes and style to the `Rating` component?

Reply:
Was Done